test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -196,6 +209,71 @@ private CassandraGenerators()
 
     }
 
+    private static String humanReadableSignPrefix(RandomnessSource rnd)
```

Comment:
this entire class is not really related to LIMIT/PER PARTITION LIMIT and me just going crazy <_<